eclass/llvm.org.eclass

```diff
@@ -489,4 +489,123 @@ llvm_install_manpages() {
 	fi
 }
 
+# @FUNCTION: llvm.org_compiler_rt_pkg_setup
```

Comment:
Please don't. This will be confusing in the long run, and requiring special invocation to make things work right is much worse than duplication.

Reply:
Same regarding `llvm.org_compiler_rt_src_configure`? Should I just copy the contents directly to `pkg_setup` and `src_configure` in libgcc ebuilds?

Reply:
Yes, and we'll get going from there.

Reply:
I was wondering the same thing, and I'm not sure. Let's copy it for now, and see how things work out. Should be easy enough to strip it afterwards, and this will make the diff between ebuilds smaller.

Would be really cool if we figured out a way to run tests using `libgcc*`.